### DISCUSSION TITLE
How can use HTML tag and style in resource

### DISCUSSION THREAD

**Discussion by army-u8** [i18next/i18next#2185] [Q&A]
I'm confused ,How can I convert the following effect to use with i18n ? any one help me thanks!

-   `i18next`: "23.11.3",
- `solid-js`: "1.6.15",
-   `tailwindcss`: "3.2.7",
- 
expected result 
```jsx
<div>There is a balance of <p class="font-bold tex-tsm">100<p>. </div>
```
ui.json here
```json

 "payments": {
            "balance": "There is a balance of {{ amount}}. "
        }
```

```jsx
<div>t("ui.payments.balance",{amount:100})</div>
```

**Comment by jamuhl** [comment]
Depends....eg.

In react you use: https://react.i18next.com/latest/trans-component
with loc-i18next you can: https://github.com/mthh/loc-i18next?tab=readme-ov-file#set-innerhtml-attributes

best search here for what is closest to your setup: https://www.i18next.com/overview/supported-frameworks

**Comment by army-u8** [comment]
> Depends....eg.
> 
> In react you use: https://react.i18next.com/latest/trans-component with loc-i18next you can: https://github.com/mthh/loc-i18next?tab=readme-ov-file#set-innerhtml-attributes
> 
> best search here for what is closest to your setup: https://www.i18next.com/overview/supported-frameworks

I use of solid-js , but This repository has been archived by the owner [https://github.com/mbarzda/solid-i18next](url) , Is there anything else that can solve this problem for me? Thank you.

**Reply by jamuhl** [reply]
never used solid...but should not be to difficult to solve

**Reply by army-u8** [reply]
I haven't found a good solution yet

**Comment by army-u8** [comment]
@adrai Can you help me look at this problem? thanks！

**Reply by adrai** [reply]
not familiar with solid... but if the repo is archived you could fork it and adapt it to your needs
